webpack.config.test.js

```diff
@@ -48,15 +40,16 @@ module.exports = {
         watchFiles: ['src/**/*', 'examples/src/**/*'],
     },
     plugins: [
-        new RemoveEmptyScriptsPlugin({}),
-        new MiniCssExtractPlugin({
-            filename: '[name].css',
```

Comment:
A CSS output filename must be specified  in the `css.filename` option of the HtmlBundlerPlugin.